Polish Translation

Patriot99 opened the pull request.

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+        },
+        "tray": {
+            "notification_title": "Drop odebrany",
+            "minimize": "Zminimalizuj do zasobnika systemowego",
```
```suggestion
            "minimize": "Minimalizuj",
```

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "name": "Status WebSocket",
+            "websocket": "WebSocket #{id}:",
+            "initializing": "Inicjalizacja...",
+            "connected": "Połączony",
+            "disconnected": "Rozłączony",
```
```suggestion
            "connected": "Połączono",
            "disconnected": "Rozłączono",
```

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+                "name": "Filtr",
+                "show": "Pokaż:",
+                "linked": "Tylko połączone",
+                "upcoming": "Najbliższe",
```
```suggestion
                "upcoming": "Nadchodzące",
```

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+                "refresh": "Odśwież"
+            },
+            "status": {
+                "linked": "Połączona ✔",
+                "not_linked": "Niepołączona ❌",
+                "active": "Aktywna ✔",
+                "upcoming": "Najbliższa ⏳",
+                "expired": "Wygasła ❌",
+                "claimed": "Odebrana ✔",
+                "ready_to_claim": "Gotowa do odebrania ⏳"
```
```suggestion
                "linked": "Połączono ✔",
                "not_linked": "Niepołączono ❌",
                "active": "Aktywna ✔",
                "upcoming": "Nadchodząca ⏳",
                "expired": "Wygasła ❌",
                "claimed": "Odebrano ✔",
                "ready_to_claim": "Gotowe do odebrania ⏳"
```

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "general": {
+                "name": "Ogólne",
+                "autostart": "Autostart: ",
+                "tray": "Autostart w zasobniku systemowym: ",
```
```suggestion
                "tray": "Autostart z zasobnika: ",
```

DevilXD commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+                "campaigns": "Zobacz wszystkie kampanie na Twitchu"
+            },
+            "how_it_works": "Jak to działa?",
+            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o ...
```
```suggestion
            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o zmianie statusu kanałów, oraz aktualizuje aktualną ilości widzów.",
```

DevilXD reviewed (changes requested):
Hello o/

You know, it's funny to me that this is like, the only other translation beyond English I could've made myself :sweat_smile: But I guess this also allows me to maintain this one going forward, so that's a plus.

First of all, the events have to be referred to as "it", not a "he". It's generally best to refer as "it" to things that have already happened (like "connected" or "linked").

You can probably just click on "Apply suggestion" on most of these. Some of the suggestions span multiple entries, so if you'd like to split the discussion on any of these, just let me know.

Let me know what you think about these overall.

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+        "no_connection": "Nie można połączyć się z Twitchem. Spróbuj ponownie za {seconds} s...."
+    },
+    "gui": {
+        "output": "Status dropienia",
```
Log is the English equivalent. I propose Dziennik zdarzeń.

Patriot99 replied:
```suggestion
        "output": "Dziennik zdarzeń",
```

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "no_token": "Nie znaleziono tokena autoryzacyjnego.",
+            "closed_window": "Okno przeglądarki Chrome zostało zamknięte przed zakończeniem procesu logowania."
+        },
+        "error_code": "Kod błędu przy logowaniu: {error_code}",
+        "unexpected_content": "Nieoczekiwany błąd zawartości, zwykle z powodu przekierowania.\nJeśli do zapewnienia dostępu do Internetu wymagane jest logowanie.",
```
I translated from German to Polish. Your suggestion fits better.

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "how_it_works": "Jak to działa?",
+            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o ...
+            "getting_started": "Pierwsze kroki",
+            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię tylko dropienie wszystkiego, odznacz „Tylko priorytet” i naciśnij na „Przeładuj”.\n• Jeśli chcesz wydropić konkretne gry, użyj listy „Priorytet”, aby ustawić uporządkowaną listę wybranych przez siebie gier. Gry ...
```
Good suggestion but punctuation is missing. 
The missing commas need to be added.
The word "aby" is a conjunction so there must be a comma (in Polish).

> prioritet -> priorytet
Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.

> I added a comma.
Jeśli interesuje cię dropienie wszystkiego, co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.

> I added a comma.
Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać.

>I added a comma.
Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.

>I added a comma.
Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.

Patriot99 replied:
```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+{
+    "status": {
+        "terminated": "\nAplikacja zatrzymana.\nZamknij okno, aby wyjść z aplikacji.",
```
Both are correct.
Adding "została" is more fitting if it is said by a program or bot.

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+        "help": {
+            "links": {
+                "name": "Pomocne linki",
+                "inventory": "Zobacz ekwipunek na Twitchu",
```
I was going to add "swój" but it is not required for the sentence to make sense.

Patriot99 replied:
```suggestion
                "inventory": "Zobacz swój ekwipunek na Twitchu",
```

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "ends": "Koniec: {time}",
+            "allowed_channels": "Dozwolone kanały:",
+            "all_channels": "Wszystkie kanały",
+            "and_more": "{amount} i więcej...",
```
I don't know what "{amount} and more" is.
My sentence says the {amount} and more ({amount} i więcej) so it should be correct.
Correct me if this string does something different.

Patriot99 replied:
```suggestion
            "and_more": "{amount} i więcej...",
```

Patriot99 commented:
> Hello o/
> 
> You know, it's funny to me that this is like, the only other translation beyond English I could've made myself 😅 But I guess this also allows me to maintain this one going forward, so that's a plus.
> 
> First of all, the events have to be referred to as "it", not a "he". It's generally best to refer as "it" to things that have already happened (like "connected" or "linked").

Hello,
Thanks for the suggestions.
I've checked them all and made changes where the translation fit better.
I also added my suggestions and changed your suggestions or the words to fit better in the Polish version. I will be 100% sure when I see the translation in the GUI.

In this case, I was translating from German to Polish.
I focused on making the program address the user as BOT.
In Polish, many sentences specify gender. 
E.g: `Odebrany drop = Received drop` or `Odebrano drop = Received drop`.
Note `Odebrany` and `Odebrano` - the difference is at the end of the word in O or Y and means the same thing: `Received`.
Both sentences indicate that the drop has already been received (past tense).
When translating in Polish, gender should not be specified so that the sentence fits any situation. In case you can't find an equivalent that doesn't specify gender you should use the male equivalent.

The male equivalent is used for Linked, since Linked means that the user has linked a Twitch account.
The word `"połączony"` is used correctly because it means that the user is connected. You can use the word `"Połączona" (if a woman is connected)` or `"Połączony" (if a man is connected)`. You can use `Połączono to not specify gender` but the word does not always fit the situation.

`Connected = Połączony, Połączona, Połączono (all are correct - depends on the case)`

Patriot99 reviewed (commented):
All requests were checked and verified.

DevilXD commented:
> In case you can't find an equivalent that doesn't specify gender you should use the male equivalent.

That can be correct in some cases, but when you consider the tenses specifically, you'll realize that "Połączony" implies a "right now" state of the situation, while "Połączono" implies an "in the past" state of the situation. This isn't even about the gendering here, more about the reflection of the current state. If something happened already (the account got connected), then "Połączono" is more fitting. "Połączony" is also valid (since the account is still connected) as you said, but since everything else already uses past tense, it'd just stand out too much, I think.

DevilXD commented on lang/Polski.json:
```diff
@@ -1,14 +1,14 @@
+        "claimed_drop": "Odebrano drop: {drop}",
+        "claimed_points": "Odebrano punkty: {points}",
+        "earned_points": "Zdobyto punkty za oglądanie: {points} | Łącznie: {balance}",
+        "no_channel": "Brak możliwych kanałów do ogłądania. Oczekiwanie na nową kampanię...",
```
```suggestion
        "no_channel": "Brak możliwych kanałów do oglądania. Oczekiwanie na nową kampanię...",
```
A simple typo fix.

DevilXD commented:
Regarding the "and X more" translation, this is how it's used in the English version:
![picture](https://user-images.githubusercontent.com/4180725/233472898-29b42980-4cab-414c-ab5b-0d2f613f9e3e.png)

I'm not sure if German has it in the wrong order or not, but it only really makes sense as a direct translation of "i X więcej", as "X i więcej" would be simply a "specific number and more", which is a weird thing to find at the end of a list containing channel names. The number isn't a channel name, only means how many more of the channels the list contains.

DevilXD commented on lang/Polski.json:
```diff
@@ -108,53 +108,53 @@
+            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o ...
             "getting_started": "Pierwsze kroki",
-            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię tylko dropienie wszystkiego, odznacz „Tylko priorytet” i naciśnij na „Przeładuj”.\n• Jeśli chcesz wydropić konkretne gry, użyj listy „Priorytet”, aby ustawić uporządkowaną listę wybranych przez siebie gier. Gry ...
+            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których  ...
```
```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```
Just a single comma too much in the third bullet point, without it it should be good.

Patriot99 replied:
There should be a comma before `"co"` so I suggest removing the next comma. The comma before `"co"` is more important than the one before `"odznacz"`. If I add a comma before `"co"` then the next one is redundant.

`Jeśli interesuje cię dropienie wszystkiego, co możliwe odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.
`

Patriot99 replied:
```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```

DevilXD reviewed (commented):
Just a few small corrections are still needed here.

Patriot99 commented:
> > In case you can't find an equivalent that doesn't specify gender you should use the male equivalent.
> 
> That can be correct in some cases, but when you consider the tenses specifically, you'll realize that "Połączony" implies a "right now" state of the situation, while "Połączono" implies an "in the past" state of the situation. This isn't even about the gendering here, more about the reflection of the current state. If something happened already (the account got connected), then "Połączono" is more fitting. "Połączony" is also valid (since the account is still connected) as you said, but since everything else already uses past tense, it'd just stand out too much, I think.

That's right. I have corrected it.

> Regarding the "and X more" translation, this is how it's used in the English version: ![picture](https://user-images.githubusercontent.com/4180725/233472898-29b42980-4cab-414c-ab5b-0d2f613f9e3e.png)
> 
> I'm not sure if German has it in the wrong order or not, but it only really makes sense as a direct translation of "i X więcej", as "X i więcej" would be simply a "specific number and more", which is a weird thing to find at the end of a list containing channel names. The number isn't a channel name, only means how many more of the channels the list contains.

Yes, I checked it in the English version and it is already corrected. 
I changed to `i {amount} więcej`.

Optionally, you can change `więcej` to `jeszcze`:

Alternatives:
`i jeszcze {amount} `
`i {amount} `

I imagine it as `i jeszcze {amount} kanałów` or `i {amount} kanałów`.
It is known that I will not add the word `"kanalów = channels"` there but wants to show here how I imagine it.

Patriot99 commented:
> > In case you can't find an equivalent that doesn't specify gender you should use the male equivalent.
> 
> That can be correct in some cases, but when you consider the tenses specifically, you'll realize that "Połączony" implies a "right now" state of the situation, while "Połączono" implies an "in the past" state of the situation. This isn't even about the gendering here, more about the reflection of the current state. If something happened already (the account got connected), then "Połączono" is more fitting. "Połączony" is also valid (since the account is still connected) as you said, but since everything else already uses past tense, it'd just stand out too much, I think.

That's right. I have corrected it.

> Regarding the "and X more" translation, this is how it's used in the English version: ![picture](https://user-images.githubusercontent.com/4180725/233472898-29b42980-4cab-414c-ab5b-0d2f613f9e3e.png)
> 
> I'm not sure if German has it in the wrong order or not, but it only really makes sense as a direct translation of "i X więcej", as "X i więcej" would be simply a "specific number and more", which is a weird thing to find at the end of a list containing channel names. The number isn't a channel name, only means how many more of the channels the list contains.

Yes, I checked it in the English version and it is already corrected. 
I changed to `i {amount} więcej`.

Optionally, you can change `więcej` to `jeszcze`:

Alternatives:
`i jeszcze {amount} `
`i {amount} `

I imagine it as `i jeszcze {amount} kanałów` or `i {amount} kanałów`.
It is known that I will not add the word `"kanalów = channels"` there but wants to show here how I imagine it.

DevilXD commented on lang/Polski.json:
```diff
@@ -154,7 +154,7 @@
             "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o ...
             "getting_started": "Pierwsze kroki",
-            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe, odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których  ...
+            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, któryc ...
```
```suggestion
            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego co możliwe, to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia, niż te niżej na liście.\n• Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n• Użyj listy „Wykluczone”, aby wskazać gry, których kampanie nigdy nie powinny być dropione.\n• Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga naciśnięcia na „Przeładuj”, aby zastosować zmiany."
```

Patriot99 replied:
```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz wydropić przedmioty.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie wszystkich przedmiotów z aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać konkretną grę lub uporządkować listę gier, z których chcesz wydropić przedmioty. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby opcjonalnie wykluczyć z dropienia gry, które nie są na liście priorytetowej - to już zależy od ciebie.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

DevilXD reviewed (changes requested):
The sentence only makes grammatical sense with a single comma in that particular place, even with the "to" addition there. Not sure why you've moved it again.

Patriot99 commented on lang/Polski.json:
```diff
@@ -0,0 +1,160 @@
+            "priority": "Priorytety",
+            "exclude": "Wykluczone",
+            "reload": "Przeładuj",
+            "reload_text": "Większość zmian wymaga przeładowania, aby natychmiast je zastosować: "
```
```suggestion
            "reload_text": "Większość zmian wymaga przeładowania, które natychmiastowo zastosuje wprowadzone zmiany: "
```

DevilXD commented on lang/Polski.json:
```diff
@@ -152,9 +152,9 @@
+            "how_it_works_text": "Co minutę aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów używane jest połączenie WebSocket, które odbiera zdarzenia o zmi ...
             "getting_started": "Pierwsze kroki",
-            "getting_started_text": "• Zaloguj się do aplikacji.\n• Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, które chcesz dropić.\n• Jeśli interesuje cię dropienie wszystkiego, co możliwe to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.\n• Jeśli chcesz otrzymać dropy z konkretnej gry, użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, któryc ...
+            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz wydropić przedmioty.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie wszystkich przedmiotów z aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać konkretną grę lub uporządkować list ...
```
This change introduces a new word: "przedmioty", and the meaning of them "coming from" a campaign. A direct translation of "przedmiot" is "item", not a "drop". The rest of the translation file refers to drops simply as a "drop", and "getting drops" is translated as "dropić" or "dropienie", which can remain valid under the circumstances that a literal translation of "zrzut" doesn't really fit in here . Using this new "przedmioty" word is a yet another inconsistency, and the simplest way to deal with it would be to go back to referring to it drops as "drop" only. Please note that each drop can have multiple "rewards" to it, and that's where the "items" translation could come in handy, at some point later on in the development, if a label to it would be ever needed.

Other than that, the priority list is for "a list of games", not "a particular game". Point #​5 doesn't need the "optionally" word there, that's like saying "an option makes things optional" which is redundant, similarly to "this depends entirely on you" at the end there - that's what all these options are for in the first place, so lets keep it short and to the point.

My suggestion implements all of the above. Because you've restructured #​2 entirely, #​2 and #​4 ended with the same sentence ending now, which isn't ideal. My suggestion is to go back with #​2 to how it was before (and finally put the comma at the right place). Everything else is fine.

```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Jeśli interesuje cię dropienie wszystkiego co możliwe, to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie z wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, które chcesz dropić. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Patriot99 replied:
I simply translated the English description. 

Point 2: Missing that linking is required in your suggestion `(Ensure your Twitch account is linked...)`. 

Point 4: `"konkretną grę" (if the user wants to add only 1 game to the list)` refers to the English `specific game (If you want to mine specific game first...)` and I added "ordered list" if the user would like to make a list of games to drop. This can be shortened to an ordered list - as you suggested. 

Point 2: What do you think about restoring the missing `"Ensure your Twitch account is linked..."`.

Suggestion - Point 2:
`Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami. Jeśli interesuje cię dropienie wszystkiego co możliwe, to odznacz „Tylko priorytetowe” i naciśnij na „Przeładuj”.`

DevilXD replied:
Hmm, actually, that's my bad for mistaking point #​2 and #​3 there. Not sure why this has happened.

Regardless, if #​2 is changed to avoid using the word "przedmioty", it becomes:

> Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.

This makes it end with almost the same words as the first sentence of point #4:

> Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, które chcesz dropić.

Regarding point #4 too, what you've said about "konkretną grę" does make some sense, however it has to be used appropriately. A user can be "interested in a particular/specific game", but then they have to add it to "a list of games". Even with one game added, it's still a list, thus you can refer to the list only as "specific games", not "a specific game or a list". Now, regarding the translation itself, the Polish language structure makes it so that there's a reversal of the word ordering there, and thus you first have to refer to setting up the list of games, and then you specify that those games are the ones you're interested in, hence the sentence I proposed does make sense. "Konkretne gry" is already included in the "które chcesz dropić" part of the sentence, so there's no need to include it anywhere else.

To resolve the similar wording of point #4, you can go back to the previous "których dropy chcesz otrzymać" translation. It's probably not the best, but at least avoids the problem.

```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie z wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, których dropy chcesz otrzymać. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Patriot99 replied:
```suggestion
            "getting_started_text": "1. Zaloguj się do aplikacji.\n2. Upewnij się, że twoje konto Twitch jest połączone ze wszystkimi kampaniami, z których chcesz dropić.\n3. Odznacz opcję „Tylko priorytetowe” i kliknij „Przeładuj”, jeśli chcesz rozpocząć dropienie ze wszystkich aktywnych kampanii.\n4. Użyj listy „Priorytety”, aby wybrać uporządkowaną listę gier, z których chcesz otrzymać dropy. Gry z górnej części listy będą miały większy priorytet dropienia niż te niżej na liście.\n5. Zaznacz opcję „Tylko priorytetowe”, aby wykluczyć z dropienia gry, które nie są na liście priorytetowej.\n6. Użyj listy „Wykluczone”, aby wskazać aplikacji gry, z których przedmioty nigdy nie powinny być dropione.\n7. Zmiana zawartości list lub opcji „Tylko priorytetowe” wymaga kliknięcia „Przeładuj”, aby aplikacja mogła zastosować wprowadzone zmiany."
```

Patriot99 commented on lang/Polski.json:
```diff
@@ -143,7 +143,7 @@
             "exclude": "Wykluczone",
             "reload": "Przeładuj",
-            "reload_text": "Większość zmian wymaga przeładowania, aby natychmiast je zastosować: "
+            "reload_text": "Większość zmian wymaga przeładowania, które natychmiastowo zastosuje wprowadzone zmiany: "
```
You simply suggested a literal translation from English. I additionally added wprowadzone. Both are correct but your suggestion is more suitable for the English version.

Patriot99 commented on lang/Polski.json:
```diff
@@ -152,9 +152,9 @@
             },
             "how_it_works": "Jak to działa?",
-            "how_it_works_text": "Co 60 sekund aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów, używane jest połączenie WebSocket, które odbiera zdarzenia o ...
+            "how_it_works_text": "Co minutę aplikacja wysyła zdarzenie „obejrzana minuta” do kanału, który jest aktualnie oglądany - to wystarczy, aby ukończyć dropienie. Zauważ, że w ten sposób możesz całkowicie pominąć konieczność pobierania rzeczywistego strumienia wideo i dźwięku. Do utrzymania statusu ONLINE lub OFFLINE kanałów używane jest połączenie WebSocket, które odbiera zdarzenia o zmi ...
```
A minute would fit more if it was equal to 60 seconds but if it is 59 seconds in the code then you are right. The change is good if it was an exact 60 seconds.

DevilXD reviewed (approved):
Everything seems to look good now.

DevilXD commented:
Thank you for your contribution :slightly_smiling_face:
